contracts/staking/StakingInfo.sol

```diff
@@ -20,6 +18,11 @@ contract IStakeManagerLocal {
         address signer;
         address contractAddress;
         Status status;
+        uint256 commissionRate;
+        uint256 lastCommissionUpdate;
+        uint256 accumulatedReward;
```

Comment:
Is it the same as `delegatorsReward`? I see two different names.

Reply:
Fixed.